cves/CVE-2013-1443.yml

```diff
@@ -290,7 +288,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The mistake that led to this vulnerability was a developer removing a security by
+    default feature. This could have been caused by unclear requirements or a miscommunication
+    in requirements, it looked as though the developer thought they were making the correct
+    modifications.
```

Comment:
The removal of security by default feature I thought that was part of the modified fix, not what led to the vulnerability?

Reply:
Added more detail to explain this, but basically the vulnerability was caused by the removal of a security by default feature, so naturally they added this feature back in to fix the vulnerability as a temporary solution.